engine/class_modules/sc_shaman.cpp

```diff
@@ -8536,6 +8628,25 @@ void shaman_t::create_buffs()
                             ->set_default_value_from_effect_type( A_ADD_PCT_MODIFIER, P_TICK_TIME )
                             ->apply_affecting_conduit( conduit.call_of_flame );
 
+  buff.fireheart = make_buff( this, "fireheart" )
```

Comment:
Consider moving this to a custom class if you don't want it to clutter up create_buffs().

Reply:
I understand, but I get the improession our shaman module is fancying creating buff like this in one big blop.